src/lib/utils.ts

```diff
@@ -34,3 +37,40 @@ export const loadScriptTag = async (srcValue: string): Promise<void> => {
 
   await loadedPromise
 }
+
+export const getCurrentContext = (): {courseId: string | undefined, host: string | undefined} => {
+  const courseId = window.M?.cfg.courseId
+  const host = window.location.host
+  return {
+    courseId,
+    host
+  }
+}
+
+function getVariantMappings(host: string | undefined, courseId: string | undefined): string {
```

Comment:
Minor: Would calling this `getVariantMapping` be more accurate since it is a singular mapping value returned?